### PR TITLE
Refactor TsTzRangeType(use PeriodFactory)

### PR DESCRIPTION
**Pull request by kumamidori**
I'm a little concerned that the same test case is in two different places `TsTzRangeTypeTest` `PeriodFactoryTest`  ... 🤔

### CODE REVIEW

File: tests/PeriodFactoryTest.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        $this->assertFalse($actual3->endsAtInclusive);
+
+        // null startsAt
+        $actual4 = $SUT->createFromString('[,"2023-01-31 23:59:59+09")');
+        $this->assertTrue($actual4->startsAtInclusive);
+        $this->assertFalse($actual4->endsAtInclusive);
+        $this->assertNull($actual4->startsAt);
+        $this->assertNotNull($actual4->endsAt);
+
+        // null startsAt
```
Comment (77web) [inline tests/PeriodFactoryTest.php:42]: :grin:
```suggestion
        // null endsAt
```

**Comment by 77web** [comment]
@kumamidori 
I think we can remove duplicated test for `TsTzRangeType` because TsTzRangeType no longer has the logic.

File: tests/PeriodFactoryTest.php (hunk 1 above)
Comment (kumamidori) [reply tests/PeriodFactoryTest.php]: Thank you, fixed! :stuck_out_tongue_closed_eyes: a8d95c4c39f0af32772e8fad15b2eeb4856b7455

**Comment by kumamidori** [comment]
@77web Thank you! 
fixed 3df7169ad6ec26f7db18d2d8d95c80e2d90c245f

**Review by 77web — APPROVED** [review]

**Pull request merged**